lldb/source/Plugins/ExpressionParser/Clang/ClangASTImporter.cpp

```diff
@@ -517,6 +520,207 @@ bool ClangASTImporter::CompleteType(const CompilerType &compiler_type) {
   return false;
 }
 
+template <class D, class O>
```

Comment:
Not your code, but I'm wondering if there's an opportunity here to add a small doxygen comment to the top of the functions?

Reply:
done